extensions/podman/src/extension.ts

```diff
@@ -337,8 +337,8 @@ export async function updateMachines(provider: extensionApi.Provider): Promise<v
 
     // Finally, we check to see if the machine that is running is set by default or not on the CLI
     // this will create a dialog that will ask the user if they wish to set the running machine as default.
-    // this should only run if we have multiple machines
-    if (machines.length > 1) {
+    // this should only run if we at least one machine
+    if (machines.length > 0) {
```

Comment:
Do we really need to keep the check ? let's just say we always check for default ?

Reply:
I think i can move it inside the else above.
The check needs to be somewhere as inside the `checkDefaultMachine` there is this code
```
let defaultMachine = machines.find(machine => machine.Default);
if (!defaultMachine) {
```
so we should update the code in any case

Reply:
I see, as you want then! thx for the explanation

Reply:
Moved above and removed the check as it is already done above 👍

Reply:
You said `moved` above, but I do not see the change in the `Files changed` ?